autoload/wiki/fzf.vim

```diff
@@ -78,14 +81,18 @@ endfunction
 "}}}1
 
 function! s:accept_page(lines) abort "{{{1
-  if len(a:lines) == 1
+  " if the query was so narrow that no page names matched, there will be two
+  " lines -- otherwise, three. The first line is the query, the second is
+  " either empty or alt-enter, depending on if enter or alt-enter was used to
+  " select, and the third line (possibly) contains the selection
+  if len(a:lines) == 2 || !empty(a:lines[1])
     let l:page = a:lines[0]
     redraw!
-    call wiki#log#info('Opening new page "' . l:page . '"')
+    call wiki#log#info('Opening page "' . l:page . '"')
```

Comment:
Why did you remove the `new` here? The point was/is to indicate that it is indeed a new page, not an old, right?

Also: What happens now if we press alt-enter and the page already exists?

Reply:
If we press alt-enter and the page already exists, the existing page will be opened and "Opening page" will be displayed. This is also the reason for removing the `new `, as we are not necessarily creating a new page.

There is a decision to make as to what happens when alt-enter is pressed and `g:wiki_map_create_page` is defined. I see two natural possibilities:

1. A page whose name is the query, verbatim, is opened/created as necessary.
2. `g:wiki_map_create_page` is first applied to the query, and the resulting page name is opened/created as necessary.

The pull request currently implements the second approach. Personally, I feel it is the less surprising of the two, and the first approach would require complicating the logic.

As for the log message, I could add a check to see if the page already exists, and, if not, print that a new page is being created. One consideration here is that we'd need to calculate the effective name of the page by possibly applying `g:wiki_map_create_page`. This would involve duplicating a few lines of `wiki#page#open`. Maybe it makes sense to move the check for whether the page exists and the resulting `wiki#log#info('Opening new page')` to `wiki#page#open` itself? At first glance, that seems like a natural place for it.

Reply:
> The pull request currently implements the second approach. Personally, I feel it is the less surprising of the two, and the first approach would require complicating the logic.

Yes, I guess you can say that; implicitly. The `g:wiki_map_create_page` is used behind the scene by `wiki#page#open`. I agree this seems to be the least surprising behaviour.

> As for the log message, ...

I agree; moving the echoing inside the `wiki#page#open` makes sense. I think it should be possible to apply the logic to the parsed url; `let l:url = wiki#url#parse()` then check the resulting `path` before `open()`.